CMake missing package sqlite

The CMake File currently didn't check if the sqlite dev package is installed.

```bash
Scanning dependencies of target sempr-core.pch_pch_target
[ 42%] Building precompiled header sempr-core.pch
[ 42%] Building CXX object src/CMakeFiles/sempr_core.dir/core/Observer.cpp.o
[ 43%] Building CXX object src/CMakeFiles/sempr_core.dir/core/EntityEvent.cpp.o
In file included from /home/ctieben/Development/sempr_pointlcoud/include/sempr/storage/ODBStorage.hpp:7:0,
                 from /home/ctieben/Development/sempr_pointlcoud/include/sempr/core/IncrementalIDGeneration.hpp:9,
                 from /home/ctieben/Development/sempr_pointlcoud/src/core/IncrementalIDGeneration.cpp:2:
/usr/include/odb/sqlite/database.hxx:10:21: fatal error: sqlite3.h: No such file or directory
compilation terminated.
src/CMakeFiles/sempr_core.dir/build.make:270: recipe for target 'src/CMakeFiles/sempr_core.dir/core/IncrementalIDGeneration.cpp.o' failed
make[2]: *** [src/CMakeFiles/sempr_core.dir/core/IncrementalIDGeneration.cpp.o] Error 1
make[2]: *** Waiting for unfinished jobs....
CMakeFiles/Makefile2:120: recipe for target 'src/CMakeFiles/sempr_core.dir/all' failed
make[1]: *** [src/CMakeFiles/sempr_core.dir/all] Error 2
make[1]: *** Waiting for unfinished jobs....
In file included from /home/ctieben/Development/sempr_pointlcoud/build/src/sempr-core.pch_all_include.cpp:41:
In file included from /home/ctieben/Development/sempr_pointlcoud/src/../include/sempr/sempr.hpp:15:
In file included from /home/ctieben/Development/sempr_pointlcoud/include/sempr/processing/DBUpdateModule.hpp:10:
In file included from /home/ctieben/Development/sempr_pointlcoud/include/sempr/query/LoadingQuery.hpp:5:
In file included from /home/ctieben/Development/sempr_pointlcoud/include/sempr/storage/ODBStorage.hpp:7:
/usr/include/odb/sqlite/database.hxx:10:10: fatal error: 'sqlite3.h' file not
      found
#include <sqlite3.h>
         ^
1 error generated.
```

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -30,6 +30,12 @@ include_directories(${PROJECT_SOURCE_DIR}/include)
 include(${ODB_USE_FILE})
 include_directories(${ODB_COMPILE_OUTPUT_DIR})
 
+# libodb-sqlite-dev does not specify a dependency to libsqlite3-dev,
+# so lets check that manually. We just need the include dir set to
+# find 'sqlite3.h'. See issue #48
+pkg_check_modules(SQLITE REQUIRED sqlite3)
+include_directories(${SQLITE_LIBRARY_DIRS})
+
 # find package eigen3
 pkg_check_modules(EIGEN REQUIRED eigen3)
 include_directories(${EIGEN_INCLUDE_DIRS})

